packages/react-composites/src/composites/ChatComposite/FileDownloadErrorBar.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IIconProps, MessageBar, MessageBarType, Stack } from '@fluentui/react';
+import React from 'react';
+
+/**
+ * @private
+ */
+export interface FileDownloadErrorBarProps {
+  /**callback to dismiss the download error message */
+  onDismissDownloadErrorMessage?: () => void;
+  /** Error message to render */
+  fileDownloadErrorMessage?: string;
+}
+
+/**
+ * @private
+ */
+export const FileDownloadErrorBar = (props: FileDownloadErrorBarProps): JSX.Element => {
+  const { fileDownloadErrorMessage, onDismissDownloadErrorMessage } = props;
+  const messageBarIconProps = (): IIconProps | undefined => {
+    const iconName = { iconName: 'ProtectedDocument' };
```

Comment:
Add a uniquely named icon to `ChatComopsiteIcons` so that Contoso can override the icon shown.

Reply:
cannot add this to ChatComopsiteIcons as it returns JSX.element, but we are passing messageBarIconProps i.e just the icon name